test/helpers/fixture.js

```diff
@@ -0,0 +1,195 @@
+export default class Fixture {
+  static pageSize() {
+    return 5;
+  }
+
+  static uspsCarrierAccountId() {
+    return 'ca_5ba7ca3632c54adeb17ad4bcac13c890';
+  }
```

Comment:
Add missing comments here
```suggestion
  // We keep the page_size of retrieving `all` records small so cassettes stay small
  static pageSize() {
    return 5;
  }

  // This is the carrier account ID for the default USPS account that comes by default. All tests should use this carrier account
  static uspsCarrierAccountId() {
    return 'ca_5ba7ca3632c54adeb17ad4bcac13c890';
  }
```